contentgrid-spring-boot-platform/build.gradle

```diff
@@ -9,9 +9,9 @@ javaPlatform {
 
 
 dependencies {
-    api platform('org.springframework.boot:spring-boot-dependencies:3.2.6')
+    api enforcedPlatform('org.springframework.boot:spring-boot-dependencies:3.2.6')
```

Comment:
Please add a comment why we enforce the platform right now, ideally referencing the PR in spring-content as well (So we can revert that once we have a new version there)